osf/utils/datetime_aware_jsonfield.py

```diff
@@ -76,19 +74,17 @@ def decode_datetime_objects(nested_value):
     return nested_value
 
 
-# TODO: This might not be necessary in Django 1.11, which allows JSONField to use
-# custom JSON encoders.
 class DateTimeAwareJSONField(JSONField):
+
+    def __init__(self, verbose_name=None, name=None, encoder=None, **kwargs):
+        encoder = DateTimeAwareJSONEncoder
```

Comment:
Drive-by: Why not just make `DateTimeAwareJSONEncoder` the default for the `encoder` kwarg, i.e. `encoder= DateTimeAwareJSONEncoder`?